src/platforms/android/migration.mdx

```diff
@@ -6,6 +6,31 @@ redirect_from:
 description: "Migrating between versions of Sentry's SDK for Android."
 ---
 
+## Migrating from `io.sentry:sentry-android 5.x` to `io.sentry:sentry-android 6.0.0`
+
+* Kotlin plugin is upgraded to `1.5`.
+* Kotlin `languageVersion` is upgraded to `1.4`.
+* `Gson` is removed as a transitive dependency and vendored in the SDK.
+  * Protocol classes now implement the `JsonSerializable` and `JsonDeserializer` interfaces.
+* `SentryOptions#shutdownTimeout` is renamed to `shutdownTimeoutMillis`.
+* Removed `@Deprecated` and `@ApiStatus.ScheduledForRemoval` methods
+  * `ITransaction#setRequest`
+  * `ITransaction#getRequest`
+  * `ITransaction#getContexts`
+  * `SentryBaseEvent#getOriginThrowable`
+  * `SentryOptions#getCacheDirSize`
+  * `SentryOptions#setCacheDirSize`
+  * `SentryOptions#isEnableSessionTracking`
+  * `SentryOptions#setEnableSessionTracking`
+* Hints changed its type from `Object` to `Map<String, Object>`
```

Comment:
Here we could give an example how users can change their code if things break? Before it was a type cast now add it to the map first and read it later when using it

Reply:
That makes sense, will do.